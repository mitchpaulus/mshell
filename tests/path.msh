`This should\U be fine` str wl
