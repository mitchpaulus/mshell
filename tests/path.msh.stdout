This should\U be fine
